README.md

```diff
@@ -118,6 +118,42 @@ following command:
 $ npm -s run extract
 ```
 
+## Hash-based URLs for contexts
+
+This is an experimental feature. We have provided redirects using identifiers
+created from the SHA-256 hashed canonicalized JSON (JCS) of the JSON-LD contexts
+in this repository at their
+[Naming things with Hashes (RFC6920)](https://www.rfc-editor.org/rfc/rfc6920.html)
+`.well-known` URLs. For example:
+
+```
+/.well-known/ni/sha-256/deb95693d486afdb1909289e399d398a4c8bceeea12f405c32359ea81c98b48a /contexts/utopia-natcert/v2.json
+```
+
+Sadly, RFC6920 does not currently support a method for expressing that JCS was
+used in the creation of these URLs, but the above absolute path would be
+consistent across domain names which implemented RFC6920 + JCS for their context
+URLs.
+
+### `Repr-Digest`
+
+In addition to the above URLs,
+[`Repr-Digest`](https://datatracker.ietf.org/doc/html/rfc9530#name-the-repr-digest-field)
```

Comment:
There's an `Unencoded-Digest` field to explore as well, that I think might avoid content-encoding issues; being used here: https://wicg.github.io/signature-based-sri/#unencoded-digest-validation-for-sri

Reply:
I'll take a look. `Repr-Digest` does not take `Content-Encoding` into account. Here's an example:
https://datatracker.ietf.org/doc/html/rfc9530#appendix-B.4-2:~:text=The%20response%20includes%20a%20Content%2DEncoding%3A%20br%20that%20indicates%20the%20selected%20representation%20is%20Brotli%2Dencoded.%20The%20Repr%2DDigest%20field%2Dvalue%20is%20therefore%20different%20compared%20to%20the%20request.